frame/elections/src/lib.rs

```diff
@@ -266,7 +266,7 @@ decl_storage! {
 
 		/// Who is able to vote for whom. Value is the fund-holding account, key is the
 		/// vote-transaction-sending account.
-		pub Proxy get(fn proxy): map hasher(blake2_256) T::AccountId => Option<T::AccountId>;
+		pub Proxy get(fn proxy): map hasher(blake2_128_concat) T::AccountId => Option<T::AccountId>;
```

Comment:
This storage item is unused in the pallet. What am I missing?

Reply:
It's not actually unused (search `Self::proxy()`) but it's never written to. Should put an issue up for it.

Reply:
Created https://github.com/paritytech/substrate/issues/5261